app/src/main/java/com/example/plantsapp/presentation/ui/tasks/TasksViewModel.kt

```diff
@@ -32,16 +33,28 @@ class TasksViewModel @AssistedInject constructor(
         MutableLiveData()
     val plantsWithTasks: LiveData<List<Pair<Plant, List<TaskWithState>>>> = _plantsWithTasks
 
+    private val _isLoading: MutableLiveData<Boolean> = MutableLiveData(false)
+    val isLoading: LiveData<Boolean> get() = _isLoading
+
     private val _launchCamera: MutableLiveData<Event<Unit>> = MutableLiveData()
     val launchCamera: LiveData<Event<Unit>> = _launchCamera
     private var takingPhotoTaskWithPlant: Pair<Plant, Task>? = null
 
     init {
+        @Suppress("TooGenericExceptionCaught")
```

Comment:
Is there any possibility to put annotation on whole class instead of one method?